### ISSUE TITLE
Training Guidance

### ISSUE THREAD

**Issue by ZhouCX117** [HLinChen/GNeSF#7]
Thanks for your excellent work! Could you please give more detailed guidance? Do I need to fine-tune the segmentation model on the scannet dataset first? I don't have scene0000_00/label-filt files, and I think it is for the segmentation map. But I don't know how to obtain them.

**Comment by HLinChen** [comment]
I have released my pretrained model [https://drive.google.com/file/d/1CGwrc31S4aI8_01JiDWi1nX9t2AzShtC/view] on the scannet. You can use it directly.

**Comment by ZhouCX117** [comment]
@HLinChen Thanks for your reply. Does it mean that I can use the pretrained segmentation checkpoint to generate 'label-fit' file for next training by 'test_seg2d.sh'?

**Comment by HLinChen** [comment]
The 'label-fit' file is provided from ScanNet dataset. I think you didn't download the whole dataset.

**Comment by HLinChen** [comment]
<img width="766" alt="image" src="https://github.com/HLinChen/GNeSF/assets/37725241/3398a3f9-301d-4abc-bbba-984fe114144c">
